data-loader/server.py

```diff
@@ -0,0 +1,56 @@
+from flask import Flask
+from app import controller
+import re
+import os
+import datetime
+
+facebook_data_root = "../data/derived/fib_results/facebook/"
+twitter_data_root = "../data/derived/fib_results/twitter/"
+PLATFORMS = ["Facebook", "Twitter"]
+LOAD_PAST_MONTH_DATA = ['2023_01']
+
+def load_current_data():
```

Comment:
It doesn't look like this function is used in this script. Can we remove it? If you need it for something else, I would either put it in that script or create a separate .py file to hold these functions.

Reply:
This function is used to get the current year month along with the directory. so in here, the it will pass the platform, current month directory to controller. The load_past_months_data function is used to add the past months data (ex- 2022_01, 2022_02, etc ) to the database.